pkg/vm/vm.go

```diff
@@ -624,14 +624,23 @@ func EncodeMemory(memory []*f.Element) []byte {
 }
 
 func DecodeMemory(content []byte) []*f.Element {
```

Comment:
Could you add a function description, along the same lines of all the function descriptions we have been doing